src/Agent.Listener/Configuration/AutoLogonRegistryManager.cs

```diff
@@ -286,7 +286,7 @@ private void UpdateUserSpecificRegistrySettings(CommandSettings command, string
 
         private void UpdateScreenSaverSettings(CommandSettings command, string securityId)
         {
-            if(!command.GetDisableScreenSaver())
+            if(command.GetKeepScreenSaver())
```

Comment:
Screen saver has a timer to lock the screen after a certain time period. If that time period is breached the machine will get locked otherwise the active session will work fine without any issue.

This is why I was talking about screen saver disable/enable thing in my earlier comments.